src/buildercore/core.py

```diff
@@ -111,13 +107,22 @@ def connect_aws_with_stack(stackname, service):
     return connect_aws_with_pname(pname, service)
 
 def find_ec2_instance(stackname):
-    "returns ec2 instance data for a *specific* stackname"
+    "returns list of ec2 instances data for a *specific* stackname"
```

Comment:
this comment is misleading if the instance isn't part of a cluster - it's not going to return a list.

`"returns ec2 instance data for a specific stackname, or a list of ec2 instance data if clustered"`